cmds/webboot/utils.go

```diff
@@ -66,3 +71,28 @@ func download(URL, fPath string) error {
 	verbose("%q is downloaded at %q\n", URL, fPath)
 	return nil
 }
+
+// getCachedDirectory recognizes the usb stick that contains the cached directory from block devices,
```

Comment:
I think either in this comment or in the README you'd want to explain the expected file structure on the USB stick so that someone using webboot would understand why this is the way it is.